tests/templates/kuttl/smoke/20-install-zk.yaml.j2

```diff
@@ -19,8 +19,8 @@ spec:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
       resources:
         cpu:
-          min: '300m'
-          max: '600m'
+          min: '100m'
+          max: '500m'
         memory:
           limit: '512Mi'
```

Comment:
WDYT of using the default resources (for a smoke test) here, so that we implicitly test if the defaults of zk are enough?

Reply:
fine with that https://github.com/stackabletech/hdfs-operator/pull/359/commits/539f10f239eafeecdaf955ca887c89717ebd7b04